spec/mention_spec.cr

```diff
@@ -0,0 +1,38 @@
+require "./spec_helper"
+
+describe Discord::Mention do
+  string = "foo bar <@123> <@!123> <@&123> <#123> <:foo:123> <a:bar:123> @everyone @here"
+
+  describe ".parse" do
+    it "parses mentions" do
+      parsed = Discord::Mention.parse(string)
+
+      parsed.size.should eq 8
```

Comment:
I'd rather you just constructed the expected result array and use `parsed.should eq expected`